src/main/java/org/broadinstitute/hellbender/engine/FeatureWalker.java

```diff
@@ -55,16 +55,16 @@ protected final void onStartup() {
 
     @SuppressWarnings("unchecked")
     private void initializeDrivingFeatures() {
-        final File drivingFile = getDrivingFeatureFile();
-        final FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingFile.toPath());
+        final GATKPath drivingPath = getDrivingFeaturePath();
+        final FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingPath.toPath());
         if (isAcceptableFeatureType(codec.getFeatureType())) {
-            drivingFeatures = new FeatureDataSource<>(new FeatureInput<>(drivingFile.getAbsolutePath()), FeatureDataSource.DEFAULT_QUERY_LOOKAHEAD_BASES, null, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, referenceArguments.getReferencePath());
+            drivingFeatures = new FeatureDataSource<>(new FeatureInput<>(drivingPath.getRawInputString()), FeatureDataSource.DEFAULT_QUERY_LOOKAHEAD_BASES, null, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, referenceArguments.getReferencePath());
 
-            final FeatureInput<F> drivingFeaturesInput = new FeatureInput<>(drivingFile.getAbsolutePath(), "drivingFeatureFile");
+            final FeatureInput<F> drivingFeaturesInput = new FeatureInput<>(drivingPath.getRawInputString(), "drivingFeatureFile");
```

Comment:
Its admittedly not obvious, but file paths are still absolutized. `FeatureInput` *is-a* `GATKPath`, so it absolutizes relative paths in it's constructor as described above.

I'm trying to get rid of the intermediate `File` semantics, but it would probably be a bit less confounding  if I instead added a `FeatureInput(GATKPath)` constructor and used that here as `new FeatureInput<>(drivingPath,...)` ? Let me know what you think.